docs/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table.md

```diff
@@ -0,0 +1,178 @@
+  - *Data Access Level* determines what kind of data will the table contain due to the multitenancy functionality 
+    - _System only_: Only system records can be inserted into this table (AD_CLIENT_ID=0, AD_ORG_ID=0), for example AD_TABLE. 
+    - _System/Client_: System or client specific records can be inserted here (AD_CLIENT_ID=anything, AD_ORG_ID=0), for example AD_ROLE 
+    - _Organization_: Only client and organization specific data can be inserted into this table (AD_CLIENT_ID<>0, AD_ORG_ID<>0), for example C_INVOICE 
+    - _Client/Organization_: Only client specific data can be inserted here, however, it can belong to a specific organizations within that client or be shared among all (AD_CLIENT_ID<>0,AD_ORG_ID=anything), for example C_BPARTNER 
+    - _All_: Any combination of AD_CLIENT_ID and AD_ORG_ID can be inserted into this table. 
+
+Save this record then press the *Create columns from DB* button to create columns within the Column tab automatically.  
+
+![](/assets/developer-guide/etendo-classic/how-to-
+guides/How_to_create_a_Table-2.png){: .legacy-image-style}
+  
+Once the creation process has finished, you will be informed of the number of columns that have been added to this table.
+
+
+![](/assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-3.png){: .legacy-image-style}
+  
+Switch to *Column* tab to see all the columns (for more information, see the AD_Column table description) that were created according to their definition within the database. You can now additionally alter the properties of each column. Each column is assigned a reference (which defines the data type) depending on its name and its data type. Run *Synchronize Terminology* process (`Application Dictionary` > `Synchronize Terminology`). Two tricky details before you run it:
+
+  - Check you have defined dependencies from your module to core and any other needed module. Synchronize Terminology will link your columns to Elements in your module or in modules it depends on. If you don't declare the dependency to core, the system will create new elements for standard columns like IsActive, CreatedBy, etc. 
+  - Do not set the Elements linked to your columns, let the Synchronize terminology to do the job. This way, the process will find the appropiate elements and will set the proper column names for the standard columns (IsActive, CreatedBy, etc.). 
+
+![](/assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-4.png){: .legacy-image-style}
+
+  
+This process tries to find an existing  application element  (within the currently developed module) and thus its label, help and description and if one is not found, a new one is created. This enables a centralized translation of the application/module.
+
+![](/assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-5.png){: .legacy-image-style}
```

Comment:
this images are new or old? because if there are new, then must remove the legacy style